src/server/fileAccessor.ts

```diff
@@ -58,18 +47,19 @@ export class VsCodeFileAccessor implements FileAccessor {
     filelist: string[] = [],
   ): string[] {
     subFolder = path.normalize(subFolder);
+    console.log(`fileAccessor.tf:getFilesInFolder:subFolder: ${subFolder}`);
 
     try {
-      fs.readdirSync(subFolder).forEach((file) => {
+      fs.readdirSync(path.join(this.ourRoot, subFolder)).forEach((file) => {
```

Comment:
Just discovered an exception is raised here, looking into it. What's weird is it does not prevent the discovery to work:

```
Searching from "/home/patrick/workspaces/homeassistant/homeassistant-config"
fileAccessor.tf:getFilesInFolder:subFolder: /home/patrick/workspaces/homeassistant/homeassistant-config
Cannot find the files in folder /home/patrick/workspaces/homeassistant/homeassistant-config: Error: ENOENT: no such file or directory, scandir '/home/patrick/workspaces/homeassistant/homeassistant-config/home/patrick/workspaces/homeassistant/homeassistant-config'
Applying schemas to 22 of your configuration files...
Assigning /home/patrick/workspaces/homeassistant/homeassistant-config/automations.yaml the configuration.yaml/automation schema
Assigning /home/patrick/workspaces/homeassistant/homeassistant-config/configuration.yaml the configuration.yaml schema
Assigning /home/patrick/workspaces/homeassistant/homeassistant-config/ui-lovelace.yaml the ui-lovelace.yaml schema
Assigning /home/patrick/workspaces/homeassistant/homeassistant-config/customize.yaml the configuration.yaml/homeassistant/customize schema
Assigning /home/patrick/workspaces/homeassistant/homeassistant-config/scripts.yaml the configuration.yaml/script schema
Assigning /home/patrick/workspaces/homeassistant/homeassistant-config/ui-lovelace/views/welcome.yaml the ui-lovelace.yaml/views schema
...
```

Reply:
Got it.